File: SNS-Server/src/main/java/ru/ruslan2570/SchoolNotificationSystem/models/Message.java
package ru.ruslan2570.SchoolNotificationSystem.models;

public class Message {

    public int messageId;
    public String text;
    public String author;
    public String role;

    public Message(int messageId, String text, String author, String role){
        this.messageId = messageId;
        this.text = text;
        this.author = author;
        this.role = role;
    }
}

File: SNS-Desktop/Model/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SNS_Desktop.Model
{
	internal class Message
	{
		public int messageId { get; set; }
		public string text { get; set; }
		public string author { get; set; }
		public string role { get; set; }
	}
}
